pkg/allowedpaths/cmd.go

```diff
@@ -0,0 +1,85 @@
+package allowedpaths
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+// CommandWithLookup should be used when the full path to the command is not known and it is
+// likely to be found in PATH.
+func CommandWithLookup(name string, arg ...string) (*exec.Cmd, error) {
```

Comment:
I wonder if we need this, or if we can push everything to using `CommandContextWithLookup`

Reply:
There were a handful of places using Command instead of CommandContext and I didn't want to update them unnecessarily, given how much regression testing this change is already going to require.

But I guess we could do `CommandContextWithLookup(context.TODO()...` for those instead. And I do think that it'd be better to push us towards using contexts for commands in the future.

Reply:
Yeah. I think if we want context, we should just `context.TODO()`

Reply:
(I've gone ahead and required a context for the newest iteration of these changes)